UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventPortalStorm.cs

```diff
@@ -52,8 +53,6 @@ public override void OnEventStart()
 
 		public override void OnEventStartTimed()
 		{
-			// TODO: play Portal Storm sound.
-
 			for (int i = 0; i < Random.Range(mobCount.x, mobCount.y); i++)
```

Comment:
Thanks. Removed wrong TODO?
```suggestion
			// TODO: play Portal Storm sound.

			for (int i = 0; i < Random.Range(mobCount.x, mobCount.y); i++)
```